src/aws-manager.js

```diff
@@ -139,10 +125,172 @@ class AwsManager {
     this.reportAmiUsage = series.amiUsage.reporter(influx);
   }
 
+  async init(){
+    try {
+      let rawJson = await this.spotRequestContainer.read('internal-provisioner-data');
+      let data = JSON.parse(rawJson);
+      // This is mainly for debugging
+      fs.writeFileSync('internal-data.json', data);
```

Comment:
Should we perhaps avoid doing this in production... Or at the very least use a non-sync writefile..